hlxsites/merative2
# feat(support-landing): add Short Cards and Content Text Body blocks

## Issue

Fixes https://jira.sdlc.merative.com/browse/MERATIVE-893 and https://jira.sdlc.merative.com/browse/MERATIVE-894

## Description
- This PR adds blocks and styles needed for the new design of the Support Hub Landing as well as Product/Brand Support Landing. 

**New**

- `Short Cards` block - used for creating CTA cards for showing brand/product logos or support with graphic links.
- `Content Text Body` block - 2/3 body that displays texts, links, lists, etc.

## Design Specs

- Figma Link - https://www.figma.com/file/PgG604uUME84O2wSdLcln2/Support-Experience?type=design&node-id=457%3A19481&mode=design&t=M3Aau9H1Kft66ENo-1

## Test URLs
  
### Support Hub Landing
![Screenshot 2023-09-20 at 2 52 29 PM](https://github.com/hlxsites/merative2/assets/1815714/8456d758-f992-4ffa-8bf8-972c809cbdb1)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/support
- After (Changes from this PR): https://feat-card-cta-893--merative2--proeung.hlx.page/support

### Product Support Landing
![Screenshot 2023-09-20 at 2 51 54 PM](https://github.com/hlxsites/merative2/assets/1815714/19edd31e-511b-48bb-858c-43e1d477dc7f)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/support/curam
- After (Changes from this PR): https://feat-card-cta-893--merative2--proeung.hlx.page/support/curam

### Product Support Landing
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/support/micromedex
- After (Changes from this PR): https://feat-card-cta-893--merative2--proeung.hlx.page/support/micromedex

## Testing Instruction


- Visit the preview links above and ensure that the front-end display matches with the design.
